usefulUtilities/spawnPointZone/entityResources/entityClientScripts/spawnPointZone.js

```diff
@@ -34,6 +36,43 @@
                 print("ERROR: COULD NOT GET ZONE USER DATA");
                 return;
             }
+            if (!spawnAreaProperties.position || Object.keys(spawnAreaProperties.position).length !== NUMBER_OF_AXES 
+                || !spawnAreaProperties.dimensions || Object.keys(spawnAreaProperties.dimensions).length !== NUMBER_OF_AXES) {
```

Comment:
what do the dimensions do here exactly?

Reply:
the area where users should move to when running the script is specified by a center point and dimensions of a square area around that point so it just sets those boundaries